pages/wg/dei.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+
+The US-RSE Association is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community in the US. The US-RSE Diversity, Equity, and Inclusion (DEI) working group is for those who are interested in regular discussions regarding DEI in the RSE community. The goal of the DEI working group is to promote the values of the US-RSE DEI Mission Statement with a particular focus on promoting opportunities for DEI education to the wider community. The group meets bi-weekly via Zoom and welcomes all newcomers.
+
+To get involved with the DEI working group, visit the [`#dei-discussion`](https://usrse.slack.com/messages/dei-discussion) channel on the US-RSE slack, or contact the <a href="mailto:dei_wg@us-rse.org">DEI working group list</a>.
+
+
+# Work Products
+
+The US-RSE DEI working group has produced and hosts the following documents and events.
+
+## DEI Mission Statement
+
+The US-RSE DEI working group's first large initiative was to create a US-RSE DEI Mission Statement to integrate into the US-RSE core values. This DEI Mission Statement was published in August 2021 on the [DEI Mission Statement page]({{ site.baseurl }}/about/dei-mission). We expect this mission statement to evolve and mature as the US-RSE community grows and welcome feedback.
+
+## DEI Speaker Series
+
+The US-RSE DEI working group invites DEI speakers to present on DEI topics. In 2021, we hosted [Dr. Cherri Pancake]({{ site.url }}/events/2021/2021-03-dei-speaker-series/) as she presented her talk, "Can We Change the Face of Computing?" and [Joanne Esch and Andrea Bowen-Jones]({{ site.url }}/events/2021/2021-07-dei-speaker-series-ncwit) of the National Center for Women & Information Technology.
+
+We will continue to invite and host speakers as they are available. We welcome suggestions regarding potential speakers.
+
+## DEI Media Club
+
+The US-RSE DEI working group hosts monthly DEI Media Club discussions in which we invite participants to read, listen to, or watch the chosen monthly DEI media and come ready to discuss the content as a small group.
+
+These are announced via the [`#events`](https://usrse.slack.com/messages/events) and [`#dei-discussion`](https://usrse.slack.com/messages/dei-discussion) channels on the US-RSE slack, as well as in monthly newsletters, and are hosted virtually via Zoom.
+
+## US-RSE Spanish Webpage Translation
+
+The US-RSE DEI working group is spearheading an effort to translate the US-RSE webpage into Spanish in order to broaden accessibility to a wider audience. This is an ongoing effort. To get involved, visit the [`#dei-discussion`](https://usrse.slack.com/messages/dei-discussion) channel on the US-RSE slack.
+
+## RSE Mentorship
+
+In 2022, the US-RSE DEI working group will begin an effort to promote and support mentorship in the RSE community. We welcome suggestions on how to get this effort started and where we would make the most impact.
```

Comment:
Mentoring might be wider than just the DEI group - I could imagine the E&T group might also get involved, focusing on skill development.

Reply:
I agree - but it is something that we've discussed actually making strides towards (and we do want involvement). I left it intentionally vague in terms of what we're doing/how we're doing it so we can update with new information as we actually progress.

Reply:
ok